src/releasers/dotnet.ts

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as chalk from 'chalk';
+import {ReleasePR, ReleaseCandidate} from '../release-pr';
+
+import {ConventionalCommits} from '../conventional-commits';
+import {GitHubTag} from '../github';
+import {checkpoint, CheckpointType} from '../util/checkpoint';
+import {Update} from '../updaters/update';
+import {Commit} from '../graphql-to-commits';
+
+// Generic
+import {Changelog} from '../updaters/changelog';
+// Dotnet specific files
+import {CommonProperties} from '../updaters/dotnet/common-properties';
+import {Project} from '../updaters/dotnet/project';
+import {ReadMe} from '../updaters/dotnet/readme';
+
+export class DotNet extends ReleasePR {
+  static releaserName = 'dotnet';
+  protected async _run() {
+    if (this.snapshot) {
+      checkpoint('running as snapshot pre-release', CheckpointType.Success);
+    }
+    const latestTag: GitHubTag | undefined = await this.gh.latestTag(
```

Comment:
It's weird to me that you need to be explicit about the type in the declaration.  What is `this.gh.latestTag` returning?

Reply:
Here and in several places below

Reply:
Fixed, this is just hold over from this being my first TypeScript project, and copy-pasta between strategies.